monero-outreach-docs/translations/es/archive/rpc_pay_es.md

```diff
@@ -1,113 +1,109 @@
+**_“Pasado por alto y poco apreciado hasta el momento, RPC-Pay permite microtansacciones instantáneas y privadas, y es la nueva unidad de pago oculta más pequeña en el universo de Monero.”_ Enero 29 del 2020**  
 
-Why run a Monero public node? It costs money, but doesn’t make money—at least not directly. A recurring concern has been how to incentivize more of these nodes. RPC-Pay offers this incentivization, and much more. It gives a way to use Monero mining hashes—just the hashes—to pay for Remote Procedure Calls (RPCs) from any RPC-Pay-enabled server, such as a Monero node. Hashes received as payment are used by the server to earn income, and because of Monero’s RandomX the mining hashes can be calculated efficiently using most popular CPUs. Paying with just hashes instead of Monero itself means that RPC-Pay does not burden the Monero network or leave a record. With RPC-Pay, mining hashes become a new stealthy smallest unit of payment in the Monero universe.
-Por qué correr un nodo público de Monero? Cuesta dinero, pero no hace dinero; al menos no directamente. Una preocupación recurrente es cómo incentivar más de estos nodos. Pagos RPC ofrece este incentivo y mucho más. Nos da una manera de usar los hashes en la minería de Monero, solamente los hashes, para pagar por las llamadas de procedimiento remoto o RPC (en inglés, Remote Procedure Calls) desde cualquier servidor de Pagos RPC activado, tal como un nodo de Monero. Los hashes recibidos como pago son usados por el servidor para obtener ingresos, y gracias al RandomX de Monero, los hashes de la minería pueden ser calculados eficientemente por los CPUs más populares. Pagar solamente con hashes en vez de Monero como tal, significa que los Pagos RPC no sobrecarga la red de Monero o dejan registro alguno. Con Pagos RPC, la minería con hashes se convierte en la nueva unidad de pago más silenciosa en el universo de Monero.
+¿Por qué ejecutar un nodo público de Monero? Cuesta dinero, pero no hace dinero; al menos no directamente. Una preocupación recurrente es cómo incentivar más de estos nodos. RPC-Pay ofrece este incentivo y mucho más. Nos da una manera de usar los hashes en la minería de Monero, pero solo los hashes, para pagar por las llamadas de procedimiento remoto (RPC - Remote Procedure Calls) desde cualquier servidor habilitado para RPC-Pay, igual como un nodo de Monero. Los hashes recibidos como pago son usados por el servidor para obtener ingresos, y gracias RandomX de Monero, los hashes de la minería pueden ser calculados eficientemente por los CPUs más comunes. Pagar solamente con hashes en vez de Monero como tal, significa que RPC-Pay no sobrecarga la red ni deja registro alguno. Con RPC-Pay, minar hashes se convierte en la nueva unidad de pago oculta más pequeña en el universo de Monero.
```

Comment:
A suggestion here:

_...desde cualquier servidor habilitado para RPC-Pay, **justo como** un nodo de Monero._

It doesn't say "just as" but in meaning it's very close by. It's still a suggestion, original phrasing is still OK.

A correction:

_...para obtener ingresos, y gracias **a** RandomX de Monero,..._ or also _**y debido a RandomX de Monero,...**_

Reply:
Changed :)